package.json

```diff
@@ -87,6 +87,7 @@
     "jose": "5.1.0",
     "load-script": "2.0.0",
     "lodash": "4.17.21",
+    "lodash-es": "4.17.21",
```

Comment:
Maybe to use [babel-plugin-import](https://github.com/umijs/babel-plugin-import)? As a result, we do not need to add `lodash-es` and change the import manually for `validator`.

Before using `babel-plugin-import` (359 kB):
```
  + First Load JS shared by all                485 kB
  ├ chunks/framework-4acc41369cd80039.js     66.8 kB
  ├ chunks/main-8ab5ed92f860346b.js          39.5 kB
  ├ chunks/pages/_app-f51a15136bed4de9.js    359 kB
  ├ chunks/webpack-5cda9d50ba88c967.js       3.43 kB
  └ css/5b39c1d79862794a.css                 16 kB
```
---
add plugin for `lodash` (344 kB)
```
+ First Load JS shared by all                470 kB
  ├ chunks/framework-4acc41369cd80039.js     66.8 kB
  ├ chunks/main-8ab5ed92f860346b.js          39.5 kB
  ├ chunks/pages/_app-52f43d4e127ad9c7.js    344 kB
  ├ chunks/webpack-06fa0929164d4608.js       3.43 kB
  └ css/5b39c1d79862794a.css                 16 kB
```
---
add plugins for `lodash` and `validator` (312 kB)
```
+ First Load JS shared by all                438 kB
  ├ chunks/framework-4acc41369cd80039.js     66.8 kB
  ├ chunks/main-8ab5ed92f860346b.js          39.5 kB
  ├ chunks/pages/_app-6c57c78134395fa5.js    312 kB
  ├ chunks/webpack-70b7fd32eaef7f54.js       3.43 kB
  └ css/5b39c1d79862794a.css                 16 kB
```

359 kB => 312 kB

config for .babelrc
```json
{
  ...
  "plugins": [
    "babel-plugin-react-docgen",
    "babel-plugin-styled-components",
+    [
+     "import",
+      {
+        "libraryName": "lodash",
+        "libraryDirectory": "",
+        "camel2DashComponentName": false
+      },
+      "lodash"
+    ],
+    [
+      "import",
+      {
+        "libraryName": "validator",
+        "libraryDirectory": "lib",
+        "camel2DashComponentName": false
+      },
+      "validator"
+   ],
    [
      "babel-plugin-formatjs",
      {
        "ast": true,
        "idInterpolationPattern": "[sha512:contenthash:base64:6]"
      }
    ]
  ],
  ...
}
```